src/main/resources/api/service_openapi.yaml

```diff
@@ -155,6 +155,28 @@ paths:
         200:
           $ref: '#/components/responses/QueryResponse'
 
+  /api/v1/unique-values/{version}:
+    post:
+      summary: Returns all distinct values
```

Comment:
May as well use the same name in both the endpoint and the documentation. Either `unique` or `distinct` seems fine to me, as long as it's consistent.

Reply:
The original ticket asked for unique value,  distinct is probably more intuitive because that is the SQL call.